### ISSUE TITLE
Credit - script issued credit amounts for 2 kits instead of 1

### ISSUE THREAD

**Issue by Denise219** [tastefully-simple/cornerstone#92]
BIC C order # 672
CID 137886
Original order - 26912585
Credit order - 26914677

Original order was placed purchasing 2 buy now kits.
I issued a credit for 1 kit plus $18.00 for shipping.

I ran this script - 
USE TSApplications
GO
INSERT INTO bcom.Credits
(
    UserID,
    OrderID,
    ItemNumber,
    Quantity
)
VALUES
(   'TSIMPLE\dsmith', -- UserID - varchar(150)
    672,                -- OrderID - int
    '150450',           -- ItemNumber - varchar(10)
    1                 -- Quantity - int
)

TSN shows Part 1 retail price for 2 kits instead of 1
![image](https://user-images.githubusercontent.com/57686045/79590238-55259000-809c-11ea-8a99-c36cafa5d3a0.png)

When I view the order in myHQ:
![image](https://user-images.githubusercontent.com/57686045/79590340-7e462080-809c-11ea-80c4-734cff83c9ce.png)

Order details in myHQ:
![image](https://user-images.githubusercontent.com/57686045/79590380-9027c380-809c-11ea-8ff1-295776ca42e3.png)

Account activity summary:
![image](https://user-images.githubusercontent.com/57686045/79590449-a6358400-809c-11ea-954f-8599f6687b0b.png)

**In this scenario -  I would have thought it would apply 1 kit price and $18.00 in shipping refund and TAX for the one kit.**
BG # 272
![image](https://user-images.githubusercontent.com/57686045/79590630-ebf24c80-809c-11ea-9f93-b89d737303bf.png)

**Comment by Denise219** [comment]
4/22 still broken
I placed an order in BC for 2 buy now kits - BC # 695
I did a partial credit for one of the buy now items.
Credit was processed # 26915921

When you view the order in TSN the Part 1 Retail amount and Order totals are not correct.
Part 1 should be $149.99 and Order totals should be ($151.49) Order total should factor in tax and shipping if applicable.

BIC C refund:
![image](https://user-images.githubusercontent.com/57686045/80037262-df189300-84b8-11ea-9dfc-73b6955e093c.png)

TSN:
![image](https://user-images.githubusercontent.com/57686045/80037313-f6f01700-84b8-11ea-8f67-d8c4141d2794.png)

myHQ:
![image](https://user-images.githubusercontent.com/57686045/80037393-1be48a00-84b9-11ea-80b7-971c3d9942b8.png)
![image](https://user-images.githubusercontent.com/57686045/80037551-6534d980-84b9-11ea-9dc0-d506fb8c58a9.png)
